graphql/tests/task/queries/taskLogs-allLogs.graphql

```diff
@@ -0,0 +1,50 @@
+query {
```

Comment:
I noticed there's also a `taskLogs` folder with queries that use `query.taskLogs` resolver; are we planning to delete that folder?

Reply:
Yes, I will delete that as a follow-up once I remove `query.taskLogs`. I didn't want to delete the tests while the resolver is still available.